napari/_tests/test_multiple_viewers.py

```diff
@@ -1,7 +1,10 @@
+import pytest
+
 from napari import Viewer
 
 
-def test_multi_viewers_dont_clash(qapp):
+@pytest.mark.skip(reason="problem with clean")
+def test_multi_viewers_dont_clash(qtbot):
```

Comment:
@tlambert03 Why is this test using `Viewer` directly instead of `make_napari_viewer`? It was added in #3692

Reply:
Can't remember,  but looking at it now, my guess is to make it as similar to whatever bug we were fixing in the PR that added this test.  That is, to avoid any addition differences that make_napari_viewer might have caused. 

is it important to change it here?  Or just curious

Reply:
without skip this test, the whole test set fails because of the leaked widget.